hs_access_control/migrations/0002_auto_20150817_1150.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.contrib.auth.models import User
+from hs_access_control.models import UserAccess
+
+
+def migrate_users(apps, schema_editor):
+    # create a 'UserAccess' record for each existing user - needed for the new access control to work
+    UserAccess.objects.all().delete()
+    for u in User.objects.all():
+        ua = UserAccess(user=u, admin=u.is_superuser)
```

Comment:
Earlier in this pull request on code that is now hidden as it is updated @alvacouch and I both commented.  There is a use case to have a privilege level less than superuser that could, e.g., be handed out to graduate students curating the site. This would be an access control admin that is separate from a Django super user.  Thus the special "curator" permission (UserAccess.admin=True, User.is_superuser=False) is desired.  Eventually we would need to build an admin interface to be able to set these privileges for users.